### PR TITLE
Dependency upgrades and new simplified demo

### PR DESCRIPTION
**Pull request by LunaticMuch**
This is a combo PR that includes the following:

1. Upgrade of various dependencies
2. A brand new demo which is simplified and does not include the whole website package
3. Material UI v5

I moved the version to `1.1.0` - whilst the versioning can be debatable, I believe we're well out from a release candidate and move to semantic versioning from here.

### CODE REVIEW

**Comment by LunaticMuch** [comment]
Feels like it does not build correctly after the new merge from the upstream 😞

**Review by IvanGoncharov — CHANGES_REQUESTED** [review]
Can you please run `npm run prettier`?
Most of the changes are prettier related so it's hard to review.

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v16.14.2
```
Comment (IvanGoncharov) [inline .nvmrc:1]: I added https://github.com/shadowspawn/node-version-usage since it more widely supported format.
Please remove.

**Comment by LunaticMuch** [comment]
> @LunaticMuch I don't see a new demo only an old one removed.

There's just an [index](https://github.com/LunaticMuch/graphql-voyager/blob/feat/v1.1.0-deps-demo-combo-bumps/demo/index.html) which uses voyager as a lib against a public endpoint.

Reckon this branch is now in trouble because we made internally all upgrades which are now coming to `master` plus the demo thing. 

It was initially complicated to extract various piece, so I made a combo PR which I knew was not a good approach.... 

I can resubmit two separate PRs, one for demo with the index only if you like this, and one for material ui v5. But I need to wait for all other deps upgrades you're sending in, otherwise we keep having conflicts.

**Comment by LunaticMuch** [comment]
@IvanGoncharov is doing already deps. 
#229 is for material UI v5.

Anything else will be in other PRs

**Pull request closed without merging**